src/map/utils/mobutils.cpp

```diff
@@ -1581,6 +1582,8 @@ a mobs regen rate, magic defense, triple attack rate from a table instead of har
                 PMob->setMobMod(MOBMOD_DETECTION, sql->GetUIntData(69));
 
                 PMob->setMobMod(MOBMOD_CHARMABLE, sql->GetUIntData(70));
+                PMob->m_roamFlags   = sql->GetUIntData(79);
+                PMob->m_SuperFamily = sql->GetUIntData(80);
```

Comment:
@TiberonKalkaz Might be good to check Dynamis just to make sure everything still links correctly there after this change (since there are DEs of different families there)

Reply:
The roam flags for dyna are set when the mob spawns so they should be ok.

Reply:
I'll check into this.

I believe we should be good: 
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/ad5a33dc-bbeb-41fb-9652-1f5eddb51f11)

Reply:
Confirmed that within Dyna Beauce - all beastmen link with each other and all statues.
Additionally, beastmen link with hydra eyes and hydras link with beastment statues.

Confirmed within Dyna Bubu that different mob types link with different mob types.
Mandies > Orc statues > Cockatrice  > Efts > Rabbits > Goblins > Dhalmels > Cralwers > Quadavs > Scorps > Yags > Birbs > Ugranites
Using god mode, speed 255, and Sleepga, was able to verify in each case (except mandies, because I started with mandies and then forgot to go back) that animal to animal and beastmen to animal, and beastmen to beastmen linking alll worked.

The only oddity appears to be a different issue.
When pulling some mobs (most noted on efts on hill sides) - the initial link or direct aggro (provoke) would not always link/aggro all of the efts that were spawned.  This appeared to be due to z-axis issues.